compiler-rt/include/CMakeLists.txt

```diff
@@ -44,6 +44,7 @@ endif(COMPILER_RT_BUILD_ORC)
 if (COMPILER_RT_BUILD_PROFILE)
   set(PROFILE_HEADERS
     profile/InstrProfData.inc
+    profile/instr_prof_interface.h
```

Comment:
Can you rename the header to follow the existing CamelCase convention?

Reply:
Yes absolutely! That said, I am following the existing convention for interface headers in other runtime components. See https://github.com/llvm/llvm-project/pull/76471/files/b741a557715eb17b69fa5b4013f2efeadcb80fda#diff-acb14b664a9f7bf0598dfd6e22dad2071ac4106753612c9f0cf3ab96bfeac0afR25, https://github.com/llvm/llvm-project/pull/76471/files/b741a557715eb17b69fa5b4013f2efeadcb80fda#diff-acb14b664a9f7bf0598dfd6e22dad2071ac4106753612c9f0cf3ab96bfeac0afL32 and numerous ones here https://github.com/llvm/llvm-project/pull/76471/files/b741a557715eb17b69fa5b4013f2efeadcb80fda#diff-acb14b664a9f7bf0598dfd6e22dad2071ac4106753612c9f0cf3ab96bfeac0afR3. Could you confirm which convention I should follow?

Reply:
Good point, looks like we treat `interface` filenames with snake case as opposed to headers. Please ignore this comment.